xml/admin_software.xml

```diff
@@ -447,40 +447,62 @@
  
  <sect1 xml:id="sec.admin.software.upgrade_caasp2">
   <title>Upgrading from &productname; 2</title>
+  <warning>
+   <title>Read This Section Carefully</title>
+   <para>
+    Before executing the single steps of the upgrade procedure,
+    carefully read all information in this overview section.
+   </para>
+  </warning>
   <para>
    As &productname; is constantly developed and improved, new versions get
    released. You are strongly advised to upgrade to a supported release. These
    upgrades may involve manual intervention.
   </para>
-  <important>
-   <title>Service Window Required</title>
-   <para>
-    Upgrades may take some time, during which services may be degraded in
-    performance or completely unavailable. Please make sure to plan a service
-    window.
-   </para>
-  </important>
   <procedure xml:id="pro.admin.upgrade.procedure">
-   <title>General Upgrade Procedure</title>
+   <title>Overview of Upgrade Procedure</title>
    <step>
     <para>
-     Upgrade the &admin_node;.
```

Comment:
This is not covered by the following sections. The Salt command seems to update all nodes at once.